GoogleSignInSwift.podspec

```diff
@@ -0,0 +1,29 @@
+Pod::Spec.new do |s|
+  s.name = 'GoogleSignInSwift'
+  s.version = '0.1.0'
+  s.swift_version = '4.0'
+  s.summary = 'Adds Swift-focused support for Google Sign In.'
+  s.description = <<-DESC
+The Google Sign-In SDK allows users to sign in with their Google account from third-party apps.
```

Comment:
Let's just sync this text with the summary for now.

Reply:
Done